test/e2e/specs/editor/blocks/list.spec.js

```diff
@@ -1176,7 +1176,11 @@ test.describe( 'List', () => {
 		await pageUtils.pressKeyWithModifier( 'secondary', 'M' ); // Emulates CTRL+Shift+Alt + M => toggle code editor
 
 		// Verify no WSOD and content is proper.
-		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+		expect( await editor.getEditedPostContent() ).toBe( `<!-- wp:list -->
```

Comment:
Let's use `editor.getBlocks` now that we're recommending it.

Reply:
Hi @ellatrix, ICYMI, do you have any more inputs on this? Feel free to request my reviews in the PR!